workflows/diagnostics/fv3net/diagnostics/offline/compute.py

```diff
@@ -339,20 +338,21 @@ def main(args):
         batches, grid, predicted_vars=model.output_variables, n_jobs=args.n_jobs
     )
 
-    # save model senstivity figures: jacobian (TODO: RF feature sensitivity)
+    # save model senstivity figures- these exclude derived variables
+    base_model = model.base_model
     try:
         plot_jacobian(
-            model,
+            base_model,
```

Comment:
The code could do a check on the model type, but the models themselves aren't public fv3fit API so this would require going into private level modules and updating the diagnostics code if new models get added with these existing methods. I think it'd also be equivalent in terms of protection against future bugs, since the last block has to have a `pass` to allow for models like the `PrecipitativeModel` that have neither method implemented. i.e. the bug addressed in this PR would also  have occurred for a check against model types instead of try/except, since the model passed to the try blocks was one without either method.